.gitignore

```diff
@@ -14,6 +14,7 @@ configs/**
 !configs/config.example.yaml
 !configs/config.ci.yaml
 !configs/bonfire.example.yaml
+!configs/cdappconfig.json
```

Comment:
Why do you ignore the file and add the file to git?

Reply:
This configuration file is an example for the format that we receive from clowder. It includes the port and path for the metrics endpoint.
The application use this to dynamically set the endpoint, so if it is changed in the future for whatever is the reason, our application automatically change this too.